rllib/algorithms/algorithm_config.py

```diff
@@ -1993,14 +2021,39 @@ def debugging(
 
         return self
 
+    @ExperimentalAPI
+    def rl_module(
+        self,
+        *,
+        rl_module_class: Optional[Type] = NotProvided,
+        _enable_rl_module_api: bool = True,
```

Comment:
`= NotProvided`!, otherwise, whenever we call this method, `_enable_rl_module_api` gets secretly set to True :)

Reply:
Oh, I see. Hmm, this is kind of different from the usual behavior, but it does make sense in this case.

It would be better then to just nix the `_enable_rl_module_api` property and create a `AlgorithmConfig.rl_module_api_enabled()` `@property` instead that returns True if `rl_module_class` is not None.